lib/msal-browser/src/telemetry/BrowserPerformanceClient.ts

```diff
@@ -102,30 +135,50 @@ export class BrowserPerformanceClient
     ): InProgressPerformanceEvent {
         // Capture page visibilityState and then invoke start/end measurement
         const startPageVisibility = this.getPageVisibility();
-
         const inProgressEvent = super.startMeasurement(
             measureName,
             correlationId
         );
+        const startTime: number | undefined = supportsBrowserPerformanceNow()
+            ? window.performance.now()
+            : undefined;
```

Comment:
What do we see in our telemetry if the performance API isn't available? Do we get some sort of indication or do we toss out the data?

Reply:
If performance API isn't available, then telemetry event duration is calculated using Unix epoch, a subject to clock skew, instead of `browser.performance.now()`. No data is tossed out. There is no indication of performance API availability in telemetry though.